BoxContentSDK/BoxContentSDK/Cache/BOXRequestCache.h

```diff
@@ -0,0 +1,23 @@
+//
+//  BOXRequestCache.h
+//  BoxContentSDK
+//
+//  Created on 8/25/15.
+//  Copyright (c) 2015 Box. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "PINCache.h"
```

Comment:
Any reason why we need to add PINCache.h import in the header and not implementation?